x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/middleware.ts

```diff
@@ -45,7 +45,7 @@ export const policyDetailsMiddlewareFactory: ImmutableMiddlewareFactory<PolicyDe
 
       // Until we get the Default configuration into the Endpoint package so that the datasource has
       // the expected data structure, we will add it here manually.
-      if (!policyItem.inputs.length) {
+      if (policyItem && !policyItem.inputs.length) {
```

Comment:
checks for unit tests

Reply:
Oh... I see why this is needed. Case `id` is invalid, is that it? in that case, would the API call return success, but `.item` be undefined/null?

Reply:
that's correct